.github/workflows/nightly-rosetta-pax-build.yaml

```diff
@@ -11,7 +11,7 @@ on:
       BASE_IMAGE:
         type: string
         description: 'PAX image built by NVIDIA/JAX-Toolbox'
-        default: 'ghcr.io/nvidia/upstream-pax:latest'
+        default: 'ghcr.io/nvidia/upstream-pax:mealkit'
```

Comment:
It looks like this default won't work anymore. Can the description and default be updated?

This image would result in:
```
BASE_IMAGE_AMD64=ghcr.io/nvidia/upstream-pax:mealkit-amd64-mealkit
BASE_IMAGE_ARM64=ghcr.io/nvidia/upstream-pax:mealkit-arm64-mealkit
```

My opinion is default = '',  and the description should be updated to specify it only accepts jax-toolbox internal images that get single arch tags

Reply:
Yeap, agree. Updated the value